### ISSUE TITLE
Onboarding: Create variations of educational friction pages

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/team-titan#1061]
Let's define how we'd slow users down who may not be eligible. 

1. Applying to a job with no overlapping hiring paths at all.
2. Applying to a Fed job in a service that does not agree with the service they selected in Profile > Federal service (might need to mention interchange agreements).
3. Applying as a US Citizen (no Fed/Vet) to a Fed/Vet position.

### Optional
A. Applying to a Vet/Fed only job as a Vet without having filled out Veterans Service.
B. Applying to a Vet/Fed only job as a Fed without having filled out Federal Service.
C. Applying to a Vet/Fed only job as a Vet without being VEOA qualified - Service dates not long enough, not honorably discharged, etc.

**Comment by humancompanion** [comment]
Tim and Debbie to talk next Wednesday.

**Comment by dalenard21** [comment]
@humancompanion 

Tim and I met yesterday.  Question, aren't scenario 1 and 3 the same? Both are situations where the 'hiring paths' don't overlap/match.  Or are we missing something unique here for the U.S. citizen applying to Vet/Fed position?

**Comment by humancompanion** [comment]
I don't believe the content would be the same. Yes, they are both situations where the hiring paths don't match but, the first scenario would be a kind of catch-all default. Whereas the third scenario is a case we want to handle explicitly. It's at least an easy case to catch and measure.

From the rough data we have thus far (just one part of a large department), the most common reason for being marked ineligible is applying to a job that is only open to Feds with status and Vets (unfortunately it's not clear if it's VEOA, VRA, or both). It's easy to remove those who are neither Feds nor Vets from that equation. The harder case will be the Optional scenarios above where the applicant is a Fed or Vet, applies, and is marked ineligible. We don't yet know why that happens as often as it does (nor whether it is predominantly Feds or Vets being eliminated since the ineligibility code covers both). I'm going to work with Staffing to break the code apart but, that will take time.

**Comment by timtimtimtimofficial** [comment]
https://opm.invisionapp.com/d/main/#/projects/prototypes/14316043
@humancompanion Here’s what @dalenard21 and I worked on for the educational friction screens. Starts with the second screen (1: No overlap…). The numbering/lettering aligns with what you labeled the scenarios on this Github issue card.
